test/DotNetWorkerTests/Diagnostics/GrpcHostLoggerTests.cs

```diff
@@ -1,3 +1,6 @@
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
I'd also recommend this in separate PRs in the future :)

Reply:
Yeah, I have a script that I run before creating PRs so it adds this header to any new files I may have created. That's why it added to existing files that didn't have either. I figured separating in a different PR would be more work. :)